### PR TITLE
Test 2019_05_21_LPS_95729_maxExpansions_help_text

### PR DESCRIPTION
**Pull request by BryanEngler**

### CODE REVIEW

**Comment by BryanEngler** [comment]
ci:test:search:norebase

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -5,6 +5,7 @@ default-search-result-permission-filter-configuration-name=Default Search Result
 disabled-entry-class-names=Disabled Entry Class Names
 disabled-entry-class-names-help=Specify the entry class names for which DefaultKeywordQueryContributor will not include the description, userName, and title fields as part of the keyword search query.
 exact-match-boost=Exact Match Boost
+exact-match-boost-help=The boost value applied when search terms match a title as an exact phrase.
```
Comment (rbohl) [inline modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: We should use full sentences for our help keys. I know many of the existing keys here aren't full sentences, but I have to get in here and do a review, and change that.
How about this?
> Apply a boost value for hits with titles exactly matching the searched keyword or phrase.

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -18,6 +19,7 @@ indexing-batch-sizes=Indexing Batch Sizes
 indexing-batch-sizes-help=The number of documents to index per batch for model types that support batch indexing. Defaults to 10000. For models with large documents, decreasing this value may improve stability when executing a full reindex.
 max-buffer-size=Maximum Buffer Size
 max-expansions=Maximum Expansions
+max-expansions-help=The number of unique titles that can be matched when search terms match a title as a phrase prefix.
```
Comment (rbohl) [inline modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: Not entirely sure I understand this one. Let me know if this distorts the intended meaning:

> Specify the maximum number of unique titles to return when the searched keywords match a title as a phrase prefix.

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties (hunk 1 above)
Comment (BryanEngler) [reply modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: is "Set the boost value to be applied when search terms match a title as an exact phrase." a full sentence?

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties (hunk 2 above)
Comment (BryanEngler) [reply modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: ya, its complicated. again can we just add "Set..." to the beginning of the sentence?

**Comment by BryanEngler** [comment]
ci:test:search:177cf7b77e50603caa150ad069c639c759381a0d

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties (hunk 1 above)
Comment (rbohl) [reply modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: Yep.

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties (hunk 2 above)
Comment (rbohl) [reply modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: Sure.

**Comment by BryanEngler** [comment]
ci:test:sf

**Pull request closed without merging**